reference_implementations/aws/online/aws_sagemaker/code/inference.py

```diff
@@ -0,0 +1,94 @@
+import os
+import json
+import torch
+import logging
+from transformers import AutoTokenizer, AutoModelForSequenceClassification
+
+logging.basicConfig(level=logging.DEBUG, format="%(asctime)s - %(name)s - %(levelname)s: %(message)s")
+logger = logging.getLogger(__name__)
+
+JSON_CONTENT_TYPE = 'application/json'
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+model_name = 'Prompsit/paraphrase-bert-en'
+pytorch_model_file = "paraphrase_bert.pt"
+
+def model_fn(model_dir):
+    """Receives the model directory/name and is responsible for loading and returning the model."""
+
+
+    logger.info(f"Loading model from: {model_dir} ")
+
+    # Specify the directory
+    directory = f'{model_dir}'
```

Comment:
Mixing up double and single quotes in the same file is frowned upon in python best practices. If it's not too much trouble it'd be worth fix it, but low priority.